compiler/AST/ImportStmt.cpp

```diff
@@ -124,23 +125,48 @@ void ImportStmt::scopeResolve(ResolveScope* scope) {
       } else {
         if (sym->isImmediate() == true) {
           USR_FATAL(this,
-                    "'import' statements must refer to module symbols "
+                    "'import' statements must include a module symbol "
                     "(e.g., 'import <module>;')");
 
         } else if (sym->name != NULL) {
-          USR_FATAL_CONT(this,
-                         "'import' of non-module symbol %s",
-                         sym->name);
-          USR_FATAL_CONT(sym,  "Definition of symbol %s", sym->name);
-          USR_STOP();
+          if (isCallExpr(src) == false) {
+            // We found a symbol that wasn't a module, but the import statement
+            // wasn't looking in a path with one or more `.`s in it.  That
+            // means this symbol is already available for unqualified access
+            USR_WARN(this, "'import' for unqualified access of symbol already "
```

Comment:
I would think that this is an error more like "Can't import without naming a module"